docs/js-core-3/index.md

```diff
@@ -23,6 +23,7 @@ sidebar_label: Overview
 | [Week 1](./week-1/lesson.md)  | Debugging         |
 | [Week 2](./week-2/lesson.md)  | Fetch, AJAX, APIS |
 | [Week 3](./week-3/lesson.md)  | Scope & `this`    |
+| [Week 4](./week-4/lesson.md)  |                   |
```

Comment:
Is week 4 supposed to be in blank?

Reply:
I believe it is empty because the tech content is not organised under 4 week format yet. I created the 4th week and added PD content, please see #620 for details. In this PR, @illicitonion links the pages I created to the module.